src/routes/mouseoverTextRouter.test.js

```diff
@@ -0,0 +1,101 @@
+const request = require('supertest');
+const { jwtPayload } = require('../test');
+const { app } = require('../app');
+const {
+  mockReq,
+  createUser,
+  mongoHelper: { dbConnect, dbDisconnect, dbClearCollections, dbClearAll },
+} = require('../test');
+const MouseoverText = require('../models/mouseoverText');
+
+const agent = request.agent(app);
+
+describe('mouseoverText routes', () => {
+  let adminUser;
+  let adminToken;
+  let reqBody = {
+    ...mockReq.body,
+  };
+
+  beforeAll(async () => {
+    await dbConnect();
+    adminUser = await createUser();
+    adminToken = jwtPayload(adminUser);
+  });
+
+  beforeEach(async () => {
+    await dbClearCollections('mouseoverText');
+    reqBody = {
+      ...reqBody,
+      newMouseoverText: 'new mouseoverText',
+    };
+  });
+
+  afterAll(async () => {
+    await dbClearAll();
+    await dbDisconnect();
+  });
+
+  describe('mouseoverTextRoutes', () => {
+    it('should return 401 if authorization header is not present', async () => {
+      await agent.post('/api/mouseoverText').send(reqBody).expect(401);
+      await agent.get('/api/mouseoverText').send(reqBody).expect(401);
+      await agent.put(`/api/mouseoverText/randomId`).send(reqBody).expect(401);
+    });
+  });
+  describe('createMouseoverText route', () => {
+    it('Should return 201 if create new mouseoverText successfully', async () => {
+      const _mouseoverText = new MouseoverText();
```

Comment:
Why are you creating a new instance of `MouseoverText`? I don't see how you are using this for.